tools/update-docker-image.bash

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -ex
+
+if [[ ! "$1" =~ [0-9]+\.[0-9]+\.[0-9]+\.[0-9]{3} ]]; then
+  echo "ERROR: A full version string must be passed in"
+  exit 1
+fi
+
+version=$1
+version_short=$(echo $1 | awk -F "." '{print $1 "." $2 "." $3}')
+
+wget -O /tmp/newrelic-php5-${version}-linux-musl.tar.gz https://download.newrelic.com/php_agent/archive/${version}/newrelic-php5-${version}-linux-musl.tar.gz;
+
+sha=$(sha256sum /tmp/newrelic-php5-${version}-linux-musl.tar.gz | awk '{print $1}')
```

Comment:
Ideally we'd create the SHA sum on package creation, store it somewhere and use it here. If packages on the download site get corrupted, this cycle of SHA creation and check might still file (calculating a SHA sum of an already corrupted package).

But that's definitely out-of-scope for this work, I just wanted to note it.